test/_webrtc-mocks.js

```diff
@@ -0,0 +1,67 @@
+// RTC classes mocks
+export class RTCPeerConnectionMock {
+  constructor(options, optional) {
+    this.constructorSpy.apply(this, arguments);
+    this.localDescription = null;
+    this.remoteDescription = null;
+    this.iceCandidates = [];
+    this.sentMessages = [];
+  }
+
+  constructorSpy() {
```

Comment:
Is this method necessary?

Reply:
Removed that. Tested constructor params in different way.